rts/Lua/LuaVBOImpl.cpp

```diff
@@ -456,6 +534,25 @@ std::tuple<uint32_t, uint32_t, uint32_t> LuaVBOImpl::GetBufferSize()
 	);
 }
 
+
+/*** Uploads the data (array of floats) into the VBO
+ *
+ * @function VBO:Upload
+ * @tparam {number,...} vboData a lua array of values to upload into the
+ * VBO
```

Comment:
I think `attribIdx` is missed here.

* @number[opt=-1] attribIdx. If supplied with non-default value, when the data from vboData will only be used to upload the data to this particular attribute. The whole vboData is expected to contain only attribIdx data. Otherwise all attributes get updated sequentially across attributes and elements.

Reply:
```
 * @number[opt=0] elemOffset. Specifies VBO element to start uploading data into.
 * @number[opt=0] luaStartIndex. Start uploading from that element in supplied Lua array
 * @number[opt=auto] luaFinishIndex. Consider this element the last element in Lua array
```